src/templates/ssr/fabric/index.svelte

```diff
@@ -0,0 +1,154 @@
+<!-- https://polyfill.io/v3/polyfill.min.js?features=default -->
+<script lang="ts">
+	import { CLICK_MACRO, GAMVariable } from '$lib/gam';
+	import { isValidReplacedVariable } from '$lib/gam';
+	import AdvertisementLabel from '$templates/components/AdvertisementLabel.svelte';
+	import Pixel from '$templates/components/Pixel.svelte';
+
+	export let Trackingpixel: GAMVariable;
+	export let Researchpixel: GAMVariable;
+	export let Viewabilitypixel: GAMVariable;
+	export let ClickthroughUrl: GAMVariable;
+	export let thirdPartyJSTracking: GAMVariable;
+
+	export let Layer1BackgroundImage: GAMVariable;
+	export let Layer1BackgroundPosition: GAMVariable;
+	export let Layer2BackgroundImage: GAMVariable;
+	export let Layer2BackgroundPosition: GAMVariable;
+	export let Layer3BackgroundImage: GAMVariable;
+	export let Layer3BackgroundPosition: GAMVariable;
+
+	export let MobileLayer1BackgroundImage: GAMVariable;
+	export let MobileLayer1BackgroundPosition: GAMVariable;
+	export let MobileLayer2BackgroundImage: GAMVariable;
+	export let MobileLayer2BackgroundPosition: GAMVariable;
+	export let MobileLayer3BackgroundImage: GAMVariable;
+	export let MobileLayer3BackgroundPosition: GAMVariable;
+</script>
+
+<AdvertisementLabel />
+<div class="creative creative--fabric">
```

Comment:
can merge these classes also

Reply:
I understand your point here and probably in this file we can do that but I guess later on we might have scss files with the reusable properties and I guess creative will have its own class.